### ISSUE TITLE
Look into the stride, kernel size, and padding for nn.ConvTranspose2d

### ISSUE THREAD

**Issue by sabeenlohawala** [sabeenlohawala/tissue_labeling#44]
https://github.com/sabeenlohawala/tissue_labeling/blob/a872f62372ae9b4ae370ece1edeebe8f963f0a88/TissueLabeling/models/original_unet.py#L14C13-L15C78

look at this pytorch UNet implementation for reference:
https://github.com/milesial/Pytorch-UNet/blob/master/unet/unet_model.py
https://github.com/milesial/Pytorch-UNet/blob/master/unet/unet_parts.py

**Comment by hvgazula** [comment]
and compare with https://github.com/neuronets/nobrainer/blob/ed0d609333f9aea724c5fd45962c36dfd8a13d88/nobrainer/models/unet.py#L34
